### PR TITLE
feat: #WB-3179 #WB-3184 #WB-3229 manage positions in a structure

### PR DESCRIPTION
**Pull request by Romu-C**
# Description

First implementation for the new feature positions add managment screen for CRUD operations.

## Fixes

(Enter here Jira or Redmine ticket(s) links)

## Type of change

Please check options that are relevant.

- [ ] Chore (PATCH)
- [ ] Doc (PATCH)
- [ ] Bug fix (PATCH)
- [x] New feature (MINOR)

## Which packages changed?

Please check the name of the package you changed

- [x admin
- [ ] app-registry
- [ ] archive
- [ ] auth
- [ ] cas
- [ ] common
- [ ] communication
- [ ] conversation
- [ ] directory
- [ ] feeder
- [ ] infra
- [ ] portal
- [ ] session
- [ ] test
- [ ] tests
- [ ] timeline
- [ ] workspace

## Tests

1. Describe here the tests you performed
2. Step by step
3. With expected results

# Reminder

- Security flaws
- Performance impacts (think bulk !)
- Unit tests were replayed
- Unit tests were added and/or changed
- I have updated the reminder for the version including my modifications

- [ ] All done ! :smiley:

### CODE REVIEW

**Comment by Romu-C** [comment]
Il y a encore des mock au niveau du fichier [userPostionServices](https://github.com/edificeio/entcore/pull/501/files#diff-41b02777d813548a5c163b5960f85d38afb50c51af97200309a7a2cb89cd7242)

File: admin/src/main/ts/src/app/core/services/user-position.service.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    // const res = await http.post<UserPosition>(
+    //   `/directory/positions/${userPosition.id}`,
+    //   userPosition,
+    // );
+    // return res;
+  }
+
+  public async deleteUserPosition(id: string) {
+    const res = await http.delete<UserPosition>(
+      `/directory/positions/${id}`,
```
Comment (ccreusat) [inline admin/src/main/ts/src/app/core/services/user-position.service.ts:34]: j'aurai fait une `const` type "endpoint / baseurl` pour `const positionsURL = /directory/positions` et pas te le retaper à chaque fois :)

**Comment by Romu-C** [comment]
Il reste encore 2-3 ODO qui sont pour ajouter des toaster de notification lorsqu'on modifie/supprime/crée une fonction

**Comment by jcbe-ode** [comment]
Annulée => commits intégrés à la [PR #504](https://github.com/edificeio/entcore/pull/504)

**Pull request closed without merging**